pkg/provision/config/config.go

```diff
@@ -0,0 +1,71 @@
+//
+// Copyright (c) 2019-2021 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package config
+
+import (
+	"fmt"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/devfile/devworkspace-operator/controllers/workspace/provision"
+	"github.com/devfile/devworkspace-operator/pkg/constants"
+)
+
+const (
+	commonPVCSizeKey = "commonPVCSize"
```

Comment:
well, during testing I found more config we may want to configure, but I'm not sure.
Like pvc name. For CRW it's [claim-che-workspace](https://github.com/eclipse-che/che-server/blob/main/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties#L385), not sure if CRW expects Che Workspace and DevWorkspaces uses different PVCs.

Also, see che.infra.kubernetes.pvc.storage_class_name.

it's out of the current PR scope for sure, but WDYT?

Reply:
My idea was that the config struct could be extended to include additional information as needed (storage class makes sense). 

For common PVC name, I think it's a limited use case that would be complex to support -- e.g. what happens if it is updated after `claim-che-workspace` has been created? It seems like it would be an open door for leaking storage after workspaces are deleted, etc.